test/common.py

```diff
@@ -986,45 +971,6 @@ def require_wasm64(self):
     else:
       self.fail('either d8 or node >= 24 required to run wasm64 tests.  Use EMTEST_SKIP_WASM64 to skip')
 
-  def require_simd(self):
-    if self.is_browser_test():
-      return
-
-    nodejs = self.get_nodejs()
-    if nodejs:
-      version = shared.get_node_version(nodejs)
-      if version >= (16, 0, 0):
-        self.js_engines = [nodejs]
-        return
-
-    if config.V8_ENGINE and config.V8_ENGINE in self.js_engines:
-      self.emcc_args.append('-sENVIRONMENT=shell')
-      self.js_engines = [config.V8_ENGINE]
-      return
-
-    if 'EMTEST_SKIP_SIMD' in os.environ:
-      self.skipTest('test requires node >= 16 or d8 (and EMTEST_SKIP_SIMD is set)')
-    else:
-      self.fail('either d8 or node >= 16 required to run wasm64 tests.  Use EMTEST_SKIP_SIMD to skip')
```

Comment:
Is removing `require_simd` intentional? Then maybe it would be good to include in the PR description too.

Reply:
Oops, yes, I was undecided about whether to split that out.